src/component/CheckoutProduct.js

```diff
@@ -0,0 +1,108 @@
+import "./CheckoutProduct.css";
+import React, { useState, useEffect } from "react";
+
+import { useStateValue } from "../context/StateProvider";
+import Swal from "sweetalert2";
```

Comment:
you weren't supposed to use off the shelf components. Rather write them by hand.